README.md

```diff
@@ -44,7 +44,8 @@ Add platform to `config.json`, for configuration see below.
       "hide": ["wifi", "<ain>"],
       "concurrent": false,
       "options": {
-        "strictSSL": false
+        "strictSSL": false,
+        "wifiName: "Guest WLAN"
```

Comment:
Please move on level with concurrent- these options are passed to the http request (although thats now obvious from the name)

Reply:
changed, thanks for the info 👍